proof/refine/Invariants_H.thy

```diff
@@ -223,21 +174,7 @@ primrec live0' :: "Structures_H.kernel_object \<Rightarrow> bool" where
 
 (* hyp_refs *)
 
-definition tcb_vcpu_refs' :: "machine_word option \<Rightarrow> (obj_ref \<times> reftype) set" where
-  "tcb_vcpu_refs' t \<equiv> set_option t \<times> {TCBHypRef}"
-
-definition tcb_hyp_refs' :: "arch_tcb \<Rightarrow> (obj_ref \<times> reftype) set" where
-  "tcb_hyp_refs' t \<equiv> tcb_vcpu_refs' (AARCH64_H.atcbVCPUPtr t)"
-
-definition vcpu_tcb_refs' :: "obj_ref option \<Rightarrow> (obj_ref \<times> reftype) set" where
-  "vcpu_tcb_refs' t \<equiv> set_option t \<times> {HypTCBRef}"
-
-definition refs_of_a' :: "arch_kernel_object \<Rightarrow> (obj_ref \<times> reftype) set" where
-  "refs_of_a' x \<equiv> case x of
-     AARCH64_H.KOASIDPool asidpool \<Rightarrow> {}
-   | AARCH64_H.KOPTE pte \<Rightarrow> {}
-   | AARCH64_H.KOVCPU vcpu \<Rightarrow> vcpu_tcb_refs' (AARCH64_H.vcpuTCBPtr vcpu)"
-
+(* FIXME arch-split: do we want to expose this, or put it in Arch? *)
 definition hyp_refs_of' :: "kernel_object \<Rightarrow> (obj_ref \<times> reftype) set" where
```

Comment:
I think we're exposing it AInvs as well, so I'd keep it set up the same way as there.

Reply:
ok will remove FIXME